dissect/target/plugins/browsers/firefox.py

```diff
@@ -17,6 +17,7 @@ class FirefoxPlugin(Plugin):
         "AppData/Roaming/Mozilla/Firefox/Profiles",
         "AppData/local/Mozilla/Firefox/Profiles",
         ".mozilla/firefox",
+        "snap/firefox/common/.mozilla",
```

Comment:
```suggestion
        "snap/firefox/common/.mozilla/firefox",
```
I believe this is the directory that actually contains the profiles.